invisible_cities/cities/beersheba.py

```diff
@@ -432,6 +443,7 @@ def beersheba(files_in, file_out, compression, event_range, print_mod, detector_
                                               evtnum_collect.sink)            ,
                                     fl.fork(write_deconv    ,
                                             write_summary   ,
+                                            write_kdst_table,
                                             write_event_info))                ,
```

Comment:
We could also replace the `kdst_writer` function with 
```
write_kdst_table = fl.sink(kdst_from_df_writer(h5out))
filter_out_none  = fl.filter(lambda x: x is not None)
[...]
fl.fork(write_deconv    ,
        write_summary   ,
        ("kdst", filter_out_none, write_kdst_table),
        write_event_info))
```
which I think is the idiomatic way to do it, but I'm afraid it might be a bit cryptic. what do you think?

Reply:
It looks an elegant solution to me, but I would prefer to use the argument explicitly, namely 
```
write_kdst_table = fl.sink(kdst_from_df_writer(h5out), args = "kdst")
filter_out_none  = fl.filter(lambda x: x is not None, args = "kdst")
[...]
fl.fork(write_deconv    ,
        write_summary   ,
        (filter_out_none, write_kdst_table),
        write_event_info))
```
just to follow the same structure as the other writers. Is it good or should we use the idiomatic way?

Reply:
I think most users are more used to `args` and find the alternative more difficult to understand, so I'm ok with this even if it is slightly more verbose than necessary.